### PR TITLE
Added License

### PR DESCRIPTION
**Pull request by TheMrM**

### CODE REVIEW

**Review by Melkeydev — CHANGES_REQUESTED** [review]
nit:
missing license

File: README.md [hunk 1]
```diff
@@ -130,4 +130,27 @@ Join the [Discord server](discord.gg/melkeydevhouse)! We're always around to hel
...
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS," WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+FOR A PARTICULAR PURPOSE, AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES, OR OTHER LIABILITY, WHETHER
+IN AN ACTION OF CONTRACT, TORT, OR OTHERWISE, ARISING FROM, OUT OF, OR IN
+CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
```
Comment (Melkeydev) [inline README.md]: Typically a license is under a different file.
It is then referenced in the README.
Check out this repo's license for a good example:
https://github.com/Melkeydev/go-blueprint/blob/main/LICENSE

**Review by Melkeydev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheMrM) [reply README.md]: Update README.md + Added LICENSE file
I think the link in the README.md will be available after the PR is approved.
Correct me if I'm mistaken!

File: README.md [hunk 2]
```diff
@@ -130,4 +130,8 @@ Join the [Discord server](discord.gg/melkeydevhouse)! We're always around to hel
 
 You can also probably catch Melkey working live on [Twitch](twitch.tv/melkey)! 
 
-Pop in and say hi! ♡
\ No newline at end of file
+Pop in and say hi! ♡
+
+## License
+
+Licensed under [MIT License](https://github.com/Melkeydev/rag-stack-lambda/LICENSE)
```
Comment (Melkeydev) [inline README.md]: This is not the best way to link to the LICENSE.
./LICENSE
is all you need relative from the ReadMe.md

File: README.md [hunk 3]
```diff
@@ -130,4 +130,8 @@ Join the [Discord server](discord.gg/melkeydevhouse)! We're always around to hel
 
 You can also probably catch Melkey working live on [Twitch](twitch.tv/melkey)! 
 
-Pop in and say hi! ♡
\ No newline at end of file
+Pop in and say hi! ♡
+
+## License
+
+Licensed under [./MIT License]
```
Comment (Melkeydev) [inline README.md]: Still not quite there:
```
Licensed under [MIT License](./LICENSE)
```

how about we try something like this?

**Review by Melkeydev — CHANGES_REQUESTED** [review]

**Review by Melkeydev — APPROVED** [review]
LGTM

File: README.md (hunk 3 above)
Comment (TheMrM) [reply README.md]: Thank you for the learning opportunity!

**Pull request merged**